irohad/consensus/yac/transport/yac_pb_converters.hpp

```diff
@@ -89,10 +73,20 @@ namespace iroha {
                           &sig) { vote.hash.block_signature = sig.value; },
                   [](iroha::expected::Error<std::shared_ptr<std::string>>) {});
 
-          vote.signature.signature = *stringToBlob<iroha::sig_t::size()>(
-              pb_vote.signature().signature());
-          vote.signature.pubkey = *stringToBlob<iroha::pubkey_t::size()>(
-              pb_vote.signature().pubkey());
+          auto pubkey =
```

Comment:
I think it will look better inlined in the builder

Reply:
I agree, and this also removes redundant copy.